app/controllers/claims/support/schools/claims_controller.rb

```diff
@@ -53,11 +53,7 @@ def update
   end
 
   def draft
-    Claims::Submit.call(
-      claim: @claim,
-      claim_params: { status: :draft },
-      user: current_user,
-    )
+    Claims::Claim::SupportCreateDraft.call(claim: @claim)
```

Comment:
Theoretically, we don't need to put emphasis on `Support`, right? That's controlled by this service only being called in a support controller.

Reply:
I emphasised the support part because the service will send emails to all school users. Previsously I called it `CreateDraft` but if we would want to use this service to create a draft in another controller I'm not sure if we want to send emails to all school users.

Basically I thought this should only be used by support. But if you want I can drop the `support bit`.

Reply:
Right now, we don't have an alternative flow for creating drafts, we'd always send out an email notification to the school users that a draft has been created.

That being said, if we _do_ want to deem this `CreateDraft` action as within the scope of Support's action set, we should probably name it `Claims::Support::Claim::CreateDraft`... `<Service>[::UserGroup][::Resource]::<Action>`?